<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Адрес компании:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wrd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DOCX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Адрес почты:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Город, регион:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Сайт:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_xl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XLSX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>2</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Example1.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Введите ФИО:</string>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>btn_wrd</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>btn_xl</zorder>
  <zorder>lineEdit_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
